docs/docs/deploying-to-iis.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Deploying to Microsoft Internet Information Server (IIS)
+---
+
+Deploying to Microsoft IIS is mostly copy &amp; paste.
+After `gatsby build` copy the contents of the `public` folder into the `wwwroot` folder of your IIS web.
+
+One important and sometimes overlooked aspect is to configure caching correctly.
+The following configuration will give you a good starting point in line with Gatsby's [suggested approach to caching](/docs/caching/).
+
+Install the [IIS URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite) module if it's not already there.
+It's required to define the `outboundRules` in the `web.config` given below.
+
+Add a `web.config` file to the `static` folder inside your Gatsby solution. It will be copied unchanged to the `public` folder
+during `gatsby build`.
+
+Please note that local settings you make for your web in IIS Manager change the `web.config` file;
+you have to be careful to copy any changes to `web.config` on your server back to the version in your Gatsby `static` folder.
+
+```xml:file=static\web.config
```

Comment:
```suggestion
```xml:file=static/web.config
```

Reply:
iis is only on Windows and there is the backslash for filepaths

Reply:
Noted. I wasn't sure if it was there for a reason, or just a minor error that hadn't been caught! Will resolve.